### ISSUE TITLE
_write modifies email content

### ISSUE THREAD

**Issue by zb3** [roundcube/roundcubemail#5499]
So when the email contains the following:
```
whatever src=/whatever.js whatever
```

This is displayed as:
```
whatever src=skins/SKINNAME/whatever.js whatever
```
If the user replies to an email containing that, the reply box also contains this change.
The same thing happens with globals, for instance if the email contains `$__comm_path`, this will also get replaced.

This is done via this line in `rcmail_output_html.php`, in the function `_write`
```
$output = $this->parse_with_globals($this->fix_paths($output));
```

Tested on 1.2.2

**Comment by alecpl** [comment]
I don't think we'll be able to fix this for 1.2. It looks like it may be a big backward incompatible change.

**Comment by alecpl** [comment]
Fixed, but needs some more testing.
